build.gradle

```diff
@@ -3,17 +3,25 @@ apply from: 'versions.gradle'
 apply from: 'maven.gradle'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.21'
     repositories {
         google()
         mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.4'
+        classpath 'com.android.tools.build:gradle:7.4.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "com.github.spotbugs.snom:spotbugs-gradle-plugin:5.2.5"
     }
 }
 
+ext {
+    SpotBugsTask = com.github.spotbugs.snom.SpotBugsTask
+}
+
```

Comment:
Gradle gave me headaches when configuring Spotbugs in quality scripts, this is only workaround I found